src/bitmessagecli.py

```diff
@@ -13,18 +13,17 @@
 
 """
 
-import xmlrpclib
 import datetime
 import imghdr
-import ntpath
 import json
+import ntpath
+import os
 import socket
-import time
 import sys
-import os
-
-from bmconfigparser import BMConfigParser
+import time
+import xmlrpclib
 
+from pybitmessage.bmconfigparser import BMConfigParser
```

Comment:
I'm not sure. Maybe `from .bmconfigparser import BMConfigParser` would be better.

Reply:
No point in relative imports any more. Setting the path and cwd has been fixed to work better running from both standalone and installed locations.

Reply:
but you keep removing the parens...

Reply:
The form
```python
from pybitmessage.bmconfigparser import BMConfigParser
```
will not work for standalone use-case because python wont find the package `pybitmessage`.

Reply:
Changed in recent commit to `from bmconfigparser import BMConfigParser` which works for me under all test scenarios listed in `PULL_REQUEST_TEMPLATE.md`, which I updated as well.